src/main/java/org/opendevstack/provision/authentication/PreAuthorizeUserOrAdministrator.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.opendevstack.provision.authentication;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.springframework.security.access.prepost.PreAuthorize;
+
+/** @author Sebastian Titakis */
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+@PreAuthorize("hasAnyAuthority(#provApp.allRoles())")
+public @interface PreAuthorizeUserOrAdministrator {}
```

Comment:
Minor style comment here. It will get confusing if we ever have more roles. Better change it to PreAuthorizeAllRoles or change the `@Preauthorize` content

Reply:
@martsec yeah, agree `PreAuthorizeAllRoles` fits better here... I'll rename it... thanks for reviewing!

Reply:
@martsec @oalyman I've renamed the annotation, can you please re-approve this change?